source/includes/materials/_10-index.md.erb

```diff
@@ -13,17 +14,78 @@ Content-Type: application/json; charset=utf-8
 ```
 
 ```json
-[
-  {
-    "description": "URL: https://github.com/gocd/gocd, Branch: master",
-    "fingerprint": "03c8d2a131154436b6ef2d621c6f773837481aaea8c5c1bb9c0cb9b5bc64a2f1",
-    "type": "Git"
+{
+  "_links": {
+    "doc": {
+      "href": "https://api.gocd.org/#materials"
+    },
+    "self": {
+      "href": "https://ci.example.com/go/api/config/materials"
+    }
+  },
+  "_embedded": {
+    "materials": [
+      {
+        "type": "git",
+        "fingerprint": "458ce82419bdbe41a4007ad713a730e5e467ed4fbefb421bf58aeff3e9c36e4b",
+        "attributes": {
+          "url": "https://github.com/gocd/gocd",
+          "destination": null,
+          "filter": null,
+          "invert_filter": false,
+          "name": null,
+          "auto_update": false,
+          "branch": "master",
+          "submodule_folder": null,
+          "shallow_clone": false
+        }
+      },
+      {
+        "type": "svn",
+        "fingerprint": "6b0cd6b9181434866c555f3c3bb780e950389a456764c876d804c848efbad554",
+        "attributes": {
+          "url": "https://github.com/gocd/gocd",
+          "destination": "test",
+          "filter": null,
+          "invert_filter": false,
+          "name": null,
+          "auto_update": false,
+          "username": "admin",
+          "encrypted_password": "AES:lTFKwi5NvrlqmQ3LOQ5UQA==:Ebggz5N27w54NrhSXKIbng==",
+          "check_externals": false
+        }
+      },
+      {
+        "type": "hg",
+        "fingerprint": "f6b61bc6b33e524c2a94c5be4a4661e5f1d2b74ca089418de20de10b282e39e9",
+        "attributes": {
+          "url": "ssh://hg@bitbucket.org/example/gocd-hg",
+          "destination": "test",
+          "filter": null,
+          "invert_filter": false,
+          "name": null,
+          "auto_update": false,
+          "branch": ""
+        }
+      },
+      {
+        "type": "dependency",
+        "fingerprint": "21f7963a8eca6cced5b4f20b57b4e9cb8edaed29d683c83621a77dfb499c553d",
+        "attributes": {
+          "pipeline": "up42",
+          "stage": "up42_stage",
+          "name": "up42_material",
+          "auto_update": true
+        }
+      }
+    ]
   }
-]
+}
 ```
 
-Lists all available materials, these are materials that are present in the in `cruise-config.xml`.
+Lists all available materials, these are materials that are present in the in `cruise-config.xml` or Config Repos.
```

Comment:
` are present in the in `cruise-config.xml` or Config Repos.` -> ` are present in the `cruise-config.xml` or Config Repos.`?